SessionMessagingKit/Open Groups/OpenGroupAPI.swift

```diff
@@ -165,9 +165,10 @@ public enum OpenGroupAPI {
     private static func preparedSequence(
         requests: [any ErasedPreparedRequest],
         authMethod: AuthenticationMethod,
-        using dependencies: Dependencies
+        using dependencies: Dependencies,
+        skipAuthentication: Bool = false
```

Comment:
`using dependencies` should always be the last parameter - please put `skipAuthentication` before it